cognee/infrastructure/databases/vector/pgvector/create_db_and_tables.py

```diff
@@ -0,0 +1,15 @@
+from ...relational.ModelBase import Base
+from ..get_vector_engine import get_vector_engine, get_vectordb_config
+from sqlalchemy import text
+
+async def create_db_and_tables():
```

Comment:
If you merged with the `main`, you can see that `create_database` exists in `SqlAlchemyAdapter`. Is that something we can reuse here?

Reply:
Yes and no. By that I mean we can use create_database from SqlAlchemyAdapter as the initial step, but there is an extra step to enable pgvector for a postgres database that we have to run in an engine connection context